packages/verifiable-credentials/src/credentials.ts

```diff
@@ -88,9 +88,25 @@ export default class Credentials {
 	 *
 	 * @param {string} vcJwt
 	 */
-	async verifyCredential(vcJwt: string): Promise<unknown> {
+	async verifyCredential(vcJwt: string): Promise<any> {
 		const resolver = this.getResolver();
-		return verifyCredential(vcJwt, resolver);
+		const decodedCredential = await verifyCredential(vcJwt, resolver);
+
+		if (decodedCredential) {
+			const payload = decodedCredential.payload
+			const vc = payload.vc
+
+			if (vc.expirationDate) {
+				// Ensure credential hasn't expired
+				const now = new Date().toISOString()
```

Comment:
Method should provide an optional parameter "currentDateTime" that is used instead of new Date().toISOString() if set (to allow the client to migrate cases where the datetime is incorrect on the local computer)

Reply:
Good point.

@cmcWebCode40 Add this to the `options` as well?

Reply:
Alright will do that